src/components/VisualizeChart/index.jsx

```diff
@@ -132,7 +133,7 @@ const VisualizeChart = ({ scrollResult }) => {
           variant: 'error',
           action,
         });
-        return;
```

Comment:
'return' is unnecessary as the last statement in a function with no return value.